fluent.syntax/src/test/kotlin/org/projectfluent/syntax/processor/ProcessorTest.kt

```diff
@@ -251,5 +251,24 @@ internal class ProcessorTest {
             ),
             processor.escapeTextToLiterals(pattern)
         )
+
+        pattern.elements.clear()
+        pattern.elements.addAll(
+            arrayOf(
+                TextElement("Hi "),
+                Placeable(expression = VariableReference(Identifier("var"))),
+                TextElement("!")
+            )
+        )
+
```

Comment:
@gueroJeff @RRagnar Thank you! Can you please add an inline comment in the PR, like this:

```suggestion
// The superfluous empty string literal is a bug. See #49.
```